src/main/java/ru/skypro/homework/controller/AdsController.java

```diff
@@ -1,23 +1,155 @@
+                    @ApiResponse(responseCode = "404", description = "Not Found", content = @Content(schema = @Schema(hidden = true)))
+            }
+    )
+    public ResponseEntity<ExtendedAd> getAds(@PathVariable int id) {
+        return ResponseEntity.ok(new ExtendedAd());
+    }
+
+
+    @DeleteMapping("/{id}")
+    @Operation(
+            summary = "Удаление объявления",
+            responses = {
+                    @ApiResponse(responseCode = "204", description = "No Content", content = @Content(schema = @Schema(hidden = true))),
+                    @ApiResponse(responseCode = "401", description = "Unauthorized", content = @Content(schema = @Schema(hidden = true))),
+                    @ApiResponse(responseCode = "403", description = "Forbidden", content = @Content(schema = @Schema(hidden = true))),
+                    @ApiResponse(responseCode = "404", description = "Not Found", content = @Content(schema = @Schema(hidden = true))),
+
+            }
+    )
+    public ResponseEntity<Ad> removeAd(@PathVariable int id) {
+        return ResponseEntity.ok(new Ad());
+    }
+
+
+    @PatchMapping("/{id}")
+    @Operation(
+            summary = "Добавление объявления",
+            responses = {
+                    @ApiResponse(responseCode = "200",
+                            description = "OK",
+                            content = @Content(
+                                    mediaType = MediaType.APPLICATION_JSON_VALUE,
+                                    schema = @Schema(implementation = CreateOrUpdateAd.class)
+                            )),
+                    @ApiResponse(responseCode = "401", description = "Unauthorized", content = @Content(schema = @Schema(hidden = true))),
+                    @ApiResponse(responseCode = "403", description = "Forbidden", content = @Content(schema = @Schema(hidden = true))),
+                    @ApiResponse(responseCode = "404", description = "Not Found", content = @Content(schema = @Schema(hidden = true)))
+            }
+    )
+    public ResponseEntity<CreateOrUpdateAd> udpateAd(@PathVariable int id) {
+        return ResponseEntity.ok(new CreateOrUpdateAd());
+    }
+
+
+    @GetMapping("/me")
+    @Operation(
+            summary = "Добавление объявления",
+            responses = {
+                    @ApiResponse(responseCode = "200",
+                            description = "OK",
+                            content = @Content(
+                                    mediaType = MediaType.APPLICATION_JSON_VALUE,
+                                    schema = @Schema(implementation = CreateOrUpdateAd.class)
+                            )),
+                    @ApiResponse(responseCode = "401", description = "Unauthorized", content = @Content(schema = @Schema(hidden = true)))
+            }
+    )
+    public ResponseEntity<Ads> getAdsMe() {
+        return ResponseEntity.ok(new Ads());
+    }
+
+
+    @PostMapping("/{id}/image")
```

Comment:
```suggestion
    @PatchMapping(value = "/{id}/image", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
```
запрос мультипартовый и патчевый